.github/ISSUE_TEMPLATE/7_cherry_pick.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Request a cherry-pick.
+about: As a contributor, you would like to request that a feature be cherry-picked into a release.
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+commit(s) to merge: (Enter the shortened commit hash of the commit that has been merged to master/main. <fabfeed>)
+
+merge instructions: clean merge
+
+What is the issue: (Enter brief description of the issue)
+
+What is the fix: (What does this cherry-pick fix?)
+
+Why cherrypick: (Why should this be cherry-picked and not exist as part of a regular release?)
+
+Risk: (<Low, Medium, High>, Has this been merged into flutter/flutter:master, flutter/engine:main etc?)
+
```

Comment:
Would be interesting to have a "test coverage" question: Are you confident your fix is well-tested by automated tests.